docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -159,6 +159,9 @@ Standard TensorFlow\* operations:
 | EuclideanNorm | No |
```

Comment:
Please, update the `docs/HOWTO/Custom_Layers_Guide.md` the section `Enabling Magnetic Resonance Image Reconstruction Model` with the note that this guide is applicable for the 2021.3 release of OpenVINO and that starting from 2021.4 the OpenVINO supports this model out of the box.

Reply:
Done.